modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/components/GradientIcon.kt

```diff
@@ -56,6 +58,43 @@ fun GradientIcon(
     )
 }
 
+@Composable
+fun GradientIcon(
+    painter: Painter,
+    gradientBrush: Brush,
+    contentDescription: String? = null,
+    blendMode: BlendMode = BlendMode.SrcAtop,
+    modifier: Modifier = Modifier,
+) {
+    Icon(
+        modifier = modifier
+            .graphicsLayer(alpha = 0.99f)
+            .drawWithCache {
```

Comment:
Why `drawWithCache()` over `drawWithContent()`? There's nothing being cached here.

Reply:
This was an example that I found when I was searching about how to display gradient icons in JC. But I was able to simplified this by removing the catch part